titus-server-master/src/main/java/com/netflix/titus/master/kubernetes/pod/KubePodUtil.java

```diff
@@ -292,4 +295,17 @@ public static String selectScheduler(SchedulerConfiguration schedulerConfigurati
         }
         return schedulerName;
     }
+
+    public static String selectPriorityClassName(ApplicationSLA capacityGroupDescriptor, KubePodConfiguration configuration) {
```

Comment:
Why do we need this?

Reply:
@rostyboost said we did.
Also seems like a good idea: if we are using one kubescheduler, we don't want the possibility of a ton of "elastic" jobs skipping the line.

Right now this never happens because we have 2 different schedulers. The reserved scheduler never even sees elastic jobs. But in this new world, we want to make sure elastic jobs can flood and overwhelm the scheduler and prevent important jobs from being scheduled first.

Reply:
Makes sense. 
In the previous implementation of mixed scheduling we were routing Flex tier jobs to `reserved` which made this prioritization necessary. In this iteration, we treat any Critical tier jobs that are over their reservations also as `elastic`. Maybe that finer distinction is not yet required.